chrysom/store.go

```diff
@@ -31,10 +31,10 @@ type PushReader interface {
 
 type Pusher interface {
 	// PushItem adds the item and establishes its link to the given owner in the store.
-	PushItem(owner string, item model.Item) (PushResult, error)
+	PushItem(owner string, item model.Item,ctx context.Context) (PushResult, error)
```

Comment:
Let's follow the Go standard of having the context as the first argument as show in https://golang.org/pkg/context/
```
func DoSomething(ctx context.Context, arg Arg) error {
    ...
}
```

Reply:
changed the code as per this conversation.